napari/utils/events/evented_model.py

```diff
@@ -216,3 +216,16 @@ def __eq__(self, other) -> bool:
             return True
         else:
             return self.dict() == other
+
+    @contextmanager
+    def enums_as_values(self, asval: bool = True):
+        """Temporarily override how enums are retrieved."""
```

Comment:
Super minor, but could the kwarg be `as_vaules` instead of `asval`, and could you add a `Parameters` to the doc string